pydra/engine/helpers.py

```diff
@@ -397,7 +397,7 @@ def get_open_loop():
     return loop
 
 
-def create_pyscript(script_path, checksum, rerun=False):
+def create_pyscript(script_path, checksum, rerun=False, ind=None):
```

Comment:
any reason why this wasn't removed as per slack conversation?

i.e instead of creating a pyscript, the slurm job simply does:

```
# SBATCH prefix

python -c "from pydra.engine.utils import run_task; run_task("/path/to/pkf_file", index, True)"
```